src/inmanta/module.py

```diff
@@ -286,11 +286,21 @@ def merge_specs(mainspec: "Dict[str, List[Requirement]]", new: "List[Requirement
 
 
 class InstallMode(str, enum.Enum):
+    """
```

Comment:
Should we also add a docstring for each install mode? Currently this information is mentioned in https://github.com/inmanta/inmanta-core/blob/d0959ab1d82db4f18c8f9083bdf183bbf227fe55/src/inmanta/module.py#L374. I think it would be better to move the documentation about the different install modes from `ProjectMetadata` to `InstallMode` and add a link in the docstring of `ProjectMetadata` to the documentation of `InstallMode`. Do you agree? (We can postpone this to a new ticket if the implementation would take too much time)

Reply:
Good idea. I think it should only take a couple of minutes.